requirements/dev.txt

```diff
@@ -6,5 +6,4 @@ isort ~= 5.5
 moto ~= 1.3
 pytest ~= 5.4
 python-dotenv ~=0.12
-python-language-server ~= 0.35
```

Comment:
This development dependency was introducing a security vulnerability that has been detected by Snyk (via the `ujson` package). We have been asked by train leadership to mitigate this vulnerability.

Looking at the `python-language-server` package, it's mainly used for IDE related linting/style checks (in VSCode), and not by anything within the tests or GitHub workflows.